src/cmds/restic/cmd_snapshots.go

```diff
@@ -132,5 +148,49 @@ func runSnapshots(opts SnapshotOptions, gopts GlobalOptions, args []string) erro
 
 	tab.Write(os.Stdout)
 
+	return
+}
+
+//Snapshot provides struct to store snapshots
```

Comment:
Actually I'd prefer to use the JSON representation we're storing in the repository, it is defined on the `Snapshot` struct here (so you do not need a special struct at all): https://github.com/restic/restic/blob/master/src/restic/snapshot.go#L13-L27